.github/workflows/ci_integration.yml

```diff
@@ -1,19 +1,24 @@
 ---
-name: "CI"
+# Full CI for commits to the integration branches (main, develop, next)
+name: "CI - Integration Branch"
 on:  # yamllint disable
+  pull_request:  # TODO: remove this once verified
   push:
-  pull_request:
+    branches:
+      - "main"
+      - "develop"
+      - "next"
     paths-ignore:
```

Comment:
Does having `paths-ignore` still make sense for `ci_integration.yml`? Or should we have integration CI run on *every* commit to an integration branch, even those that only touch (for example) documentation files?

Reply:
I think we're still left with the issue that without `paths-ignore`, there's temptation to use `[skip ci]` on commits that are documentation updates only, leaving the `integration-test` check dangling.

So long as such commits aren't the last commit in a PR branch going into an integration branch, the problem is moot, but we have to assert this somehow most likely in our release management "ritual".

Reply:
I think with the improvements made in this PR in general `[skip ci]` should not need to be used if much at all, especially with sights on #1635